src/io/_Private/LegacyPHPResourceSeekableHandleTrait.php

```diff
@@ -29,4 +29,10 @@ trait LegacyPHPResourceSeekableHandleTrait implements IO\SeekableHandle {
       \fseek($this->impl, $offset);
     });
   }
+
+  final public function tell(): int {
+    using new PHPWarningSuppressor();
+
+    return \ftell($this->impl);
```

Comment:
we should start consistently dealing with errno, but doesn't need doing in this PR given we're not doing it yet in the rest of HSL IO :)